### PR TITLE
Modify README_zh.md

### PR DESCRIPTION
**Pull request by wlq1212**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by gy95** [comment]
/kind doc
keep the same with English doc [README.md](https://github.com/kubeedge/kubeedge/blob/master/README.md)

File: README_zh.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 [English](./README.md) | 简体中文
 
-KubeEdge 是一个开源的系统，可将本机容器化应用编排和管理扩展到边缘端设备。 它基于Kubernetes构建，为网络和应用程序提供核心基础架构支持，并在云端和边缘端部署应用，同步元数据。KubeEdge 还支持 **MQTT** 协议，允许开发人员编写客户逻辑，并在边缘端启用设备通信的资源约束。KubeEdge 包含云端和边缘端两部分。
+KubeEdge 是一个开源的系统，可将本机容器化应用编排和管理扩展到边缘端设备。 它基于Kubernetes构建，为云和边缘之间的网络、应用程序部署和元数据同步提供了核心基础设施支持。KubeEdge 还支持 **MQTT** 协议，允许开发人员编写客户逻辑，并在边缘端启用设备通信的资源约束。KubeEdge 包含云端和边缘端两部分。
```
Comment (fisherxu) [inline README_zh.md:12]: Thanks for the fixing, and would you like to please keep this description consistent with the English version? @wlq1212

**Comment by gy95** [comment]
not this, you can just add more materials that Chinese doc doesn't have but Engilsh version has. 
Such as the two blocks.
Thanks very much  :)
```
With KubeEdge it is easy to get and deploy existing complicated machine learning, image recognition, event processing and other high level applications to the Edge. With business logic running at the Edge, much larger volumes of data can be secured & processed locally where the data is produced. With data processed at the Edge, the responsiveness is increased dramatically and data privacy is protected.

KubeEdge is an incubation-level hosted project by the Cloud Native Computing Foundation (CNCF). KubeEdge incubation announcement by CNCF.
```

**Pull request closed without merging**